MediaBrowser.Model/Dto/UpdateUserItemDataDto.cs

```diff
@@ -0,0 +1,38 @@
+namespace MediaBrowser.Model.Dto
+{
+    /// <summary>
+    /// This is used by the api to get information about a item user data.
+    /// </summary>
+    public class UpdateUserItemDataDto : UserItemDataDto
```

Comment:
Why does this inherit `UserItemDataDto`?

Reply:
I would have used `UserItemDataDto` directly if it was possible however due to the nature of possible fields not being sent it will alter user data. So, this class extends it and makes all fields nullable to not have unintended side effects. And i think the reviewers agree so far.

Reply:
If you want i can decouple the classes by duplicating the fields i feel it's really not good code case, however if that you want it's not a big deal

Reply:
I think that would be best IMO

Reply:
Thats fine with me. Do you guys agree with this requested changes? @Shadowghost @nielsvanvelzen @crobibero seeing as you approved the previous code.

Reply:
I've only reviewed the API specification which doesn't use inheritance anyway. There wouldn't be any difference on that part if this was changed, so no objection from me.

Reply:
@Bond-009 done.